contrib/gonum/qplot/plotter_funcs.go

```
@@ -99,12 +99,107 @@ func ScatterPlotter(xyer plotter.XYer, cfg ScatterConfig) PlotterFunc {
 	}
 }
 
+// BoxPlotConfig is an optional function which
+// configures a BoxPlot after creation.
+type BoxPlotConfig func(*plot.Plot, *plotter.BoxPlot)
+
+// BotPlot returns a new PlotterFunc that plots a BoxPlot.
+func BoxPlot(w vg.Length, loc float64, values plotter.Valuer, cfg BoxPlotConfig) PlotterFunc {
+	return func(plt *plot.Plot) (plot.Plotter, error) {
+		pltr, err := plotter.NewBoxPlot(w, loc, values)
+		if err != nil {
+			return nil, err
```

Should this error be propagated rather than just returned?